tests/kit/src/electron.ts

```
@@ -1,33 +1,206 @@
 
 import { Package } from '@affine-tools/utils/workspace';
-import { expect, type Page } from '@playwright/test';
+import type { Page } from '@playwright/test';
 import fs from 'fs-extra';
 import type { ElectronApplication } from 'playwright';
 import { _electron as electron } from 'playwright';
+import treeKill from 'tree-kill';
 
-import { test as base, testResultDir } from './playwright';
+import { test as base } from './playwright';
 import { removeWithRetry } from './utils/utils';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)